test/test_utils_test.go

```diff
@@ -103,24 +101,21 @@ func WaitForTCP(timeout time.Duration, addr string) error {
 
 // stringMapPropagator propagates the list of keys across a workflow,
 // interpreting the payloads as strings.
-// BORROWED FROM 'internal' PACKAGE TESTS.
+// TODO: BORROWED FROM 'internal' PACKAGE TESTS.
+// TODO: remove code duplication.
```

Comment:
stringMapPropagator is a weird name after the refactoring. May be multiKeyPropagator?

Reply:
I so get used to it and didn't pay attention.